testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/ClientTest.java

```diff
@@ -223,6 +223,42 @@ public void updateClient() {
         assertClient(client, storedClient);
     }
 
+    @Test
+    public void updateClientWithAuthzServices() throws Exception {
+      // create a new client
+      ClientRepresentation client = new ClientRepresentation();
+      client.setClientId("my-app-oldname");
+      client.setPublicClient(false);
+      client.setBearerOnly(false);
+      client.setStandardFlowEnabled(true);
+      client.setImplicitFlowEnabled(false);
+      client.setDirectAccessGrantsEnabled(true);
+      client.setServiceAccountsEnabled(true);
+      client.setAuthorizationServicesEnabled(true);
+      client.setRedirectUris(Collections.singletonList("/"));
+      assertEquals(201, realm.clients().create(client).getStatus());
```

Comment:
It is good to close the response and also it's possible to retrieve the newly created ID with something like this:
````
Response response = realm.clients().create(client);
String clientUUID = ApiUtil.getCreatedId(response);
response.close();
````